cmd/skydive_agent/skydive_agent.go

```diff
@@ -42,12 +42,14 @@ import (
 )
 
 func usage() {
-	fmt.Printf("Usage: %s -conf <config.ini> [-h]\n", os.Args[0])
+	fmt.Printf("Usage: %s -conf <config.ini> -trace <trace.pcap> [-h]\n", os.Args[0])
 }
 
 func main() {
 	filename := flag.String("conf", "/etc/skydive/skydive.ini",
 		"Config file with all the skydive parameter.")
+	pcaptrace := flag.String("trace", "",
```

Comment:
Agree
I will do a pcap2sflow-injector external tool
That make more sense and will test the whole chain